utils/spaceship.asyncState/index.js

```diff
@@ -0,0 +1,23 @@
+export const PENDING = 'pending';
+export const SUCCESS = 'success';
+export const FAILURE = 'failure';
+
+export default asyncState = stateName => {
+  switch (stateName) {
+  case PENDING:
+  case SUCCESS:
+  case FAILURE:
+    return {
```

Comment:
what does it mean in this return data?

i can't understand what this return data means, but i think it is correct in each case

`PENDING`
```javascript
return {
      pending: true,
      success: false,
      failure: false,
      [stateName]: true
    };
```

`SUCCESS`
```javascript
return {
      pending: false,
      success: true,
      failure: false,
      [stateName]: true
    };
```

`FAILURE`
```javascript
return {
      pending: false,
      success: false,
      failure: true,
      [stateName]: true
    };
```

Reply:
I add explain to #9 🙌

Reply:
yeap , thanks for explaining!